apps/src/templates/teacherDashboard/sections.js

```diff
@@ -69,6 +70,31 @@ export function renderLoginTypeAndSharingControls(sectionId) {
   );
 }
 
+export function renderSectionTable(sectionId, loginType) {
+  const dataUrl = `/v2/sections/${sectionId}/students`;
+  const element = document.getElementById('student-table-react');
+
+  $.ajax({
+    method: 'GET',
+    url: dataUrl,
+    dataType: 'json'
+  }).done(studentData => {
+    studentData = studentData.map((section) => {
+      return {
+        ...section,
+        loginType: loginType
+      };
+    });
+    ReactDOM.render(
+      <ManageStudentsTable
+        studentData={studentData}
+        id={sectionId}
+        loginType={loginType}
```

Comment:
If we are going to be embedding loginType onto each student, should we just have ManageStudentsTable extract it from studentData (i.e. `studentData[0].loginType`) rather than also passing it in separately?

Reply:
then when we have zero students, we won't have information we need to display the table headers properly

Reply:
Ahhh. I wonder if the better option would be to have ManageStudentsTable own sticking loginType onto studentData?

Reply:
How about I only do a mapping once, and I just be really explicit about everything I need and what I want the names to be? See above